src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
+                    </Tag>
+                    <span style={{fontSize: ".85em"}}>{peersCount}</span>
+                </div>
+            )
+        })
+        const mainContent = (<Space direction="vertical">{content}</Space>)
+        return (
+            <Popover key={peerToAction.key} content={mainContent} title={null}>
+                <Button type="link" onClick={() => setUpdateGroupsVisible(peerToAction, true)}>{label}</Button>
+            </Popover>
+        )
+    }
+
     return (
-        <Container style={{paddingTop: "40px"}}>
-            <Row>
-                <Col span={24}>
-                    <Title level={4}>Peers</Title>
-                    <Paragraph>A list of all the machines in your account including their name, IP and status.</Paragraph>
-                    <Space direction="vertical" size="large" style={{ display: 'flex' }}>
-                        <Row gutter={[16, 24]}>
-                            <Col xs={24} sm={24} md={8} lg={8} xl={8} xxl={8} span={8}>
-                                {/*<Input.Search allowClear value={textToSearch} onPressEnter={searchDataTable} onSearch={searchDataTable} placeholder="Search..." onChange={onChangeTextToSearch} />*/}
-                                <Input allowClear value={textToSearch} onPressEnter={searchDataTable} placeholder="Search..." onChange={onChangeTextToSearch} />
-                            </Col>
-                            <Col xs={24} sm={24} md={11} lg={11} xl={11} xxl={11} span={11}>
-                                <Space size="middle">
-                                    <Radio.Group
-                                        options={optionsOnOff}
-                                        onChange={onChangeOnOff}
-                                        value={optionOnOff}
-                                        optionType="button"
-                                        buttonStyle="solid"
+        <>
+            <Container style={{paddingTop: "40px"}}>
+                <Row>
+                    <Col span={24}>
+                        <Title level={4}>Peers</Title>
+                        <Paragraph>A list of all the machines in your account including their name, IP and status.</Paragraph>
+                        <Space direction="vertical" size="large" style={{ display: 'flex' }}>
+                            <Row gutter={[16, 24]}>
+                                <Col xs={24} sm={24} md={8} lg={8} xl={8} xxl={8} span={8}>
+                                    {/*<Input.Search allowClear value={textToSearch} onPressEnter={searchDataTable} onSearch={searchDataTable} placeholder="Search..." onChange={onChangeTextToSearch} />*/}
+                                    <Input allowClear value={textToSearch} onPressEnter={searchDataTable} placeholder="Search..." onChange={onChangeTextToSearch} />
+                                </Col>
+                                <Col xs={24} sm={24} md={11} lg={11} xl={11} xxl={11} span={11}>
```

Comment:
Codacy has a fix for the issue: [Replace `····························` with `··············`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
              <Col xs={24} sm={24} md={11} lg={11} xl={11} xxl={11} span={11}>
```